bindings/java/src/java_module_host.c

```diff
@@ -213,8 +213,17 @@ static MODULE_HANDLE JavaModuleHost_Create(BROKER_HANDLE broker, const void* con
                                         {
                                             jmethodID jModule_constructor = JNIFunc(result->env, GetMethodID, jModule_class, CONSTRUCTOR_METHOD_NAME, MODULE_CONSTRUCTOR_DESCRIPTOR);
                                             exception = JNIFunc(result->env, ExceptionOccurred);
+                                            bool emptyConstructor = false;
                                             if (jModule_constructor == NULL || exception)
                                             {
+                                                emptyConstructor = true;
```

Comment:
this flag doesn't mean that no-args constructor was found, just that it tries to find it. If it is not found it will log the exception and return NULL.